js/controller.js

```diff
@@ -1346,6 +1331,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
   
     moveVRToDirection: function (rotate, direction) {
+      console.log('controller moveVRToDirection');
```

Comment:
For debugging we use the `OO.log()` function so that these messages don't show up on production environment.

Reply:
fix